types/jss/index.d.ts

```diff
@@ -3,118 +3,305 @@
 // Definitions by: Brenton Simpson <https://github.com/appsforartists>
 //                 Oleg Slobodskoi <https://github.com/kof>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.3
 
-export interface Rule {
-	className: string;
```

Comment:
Did you do some weird whitespace replacement as part of this diff?  Why is every line of the file appearing changed, even the ones that appear to be identical?

Reply:
Sorry, I had Prettier format-on-save enabled, will fix.

Reply:
FYI, the reason Prettier did this is because the `.editorconfig` in the base directory of `DefinitelyTyped` specifies that tabs are 4 spaces. So this typing file appears to be flouting the project's conventions 🙂

Reply:
I'm not sure where the tabs come from.  I don't typically use tabs either - I presumed it was a tslint thing, but changing to spaces doesn't seem to break the build.

I'm fine with changing to spaces in a separate PR, but let's keep this one reviewable. 😃